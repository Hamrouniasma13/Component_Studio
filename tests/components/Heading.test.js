import React from 'react';
import renderer from 'react-test-renderer';

import Heading from '../../components/Heading';

const loremTitle = 'Magna cillum consectetur.';
describe('Component: <Heading />', () => {
  it('render <h1>', () => {
    const tree = renderer.create(<Heading title={loremTitle} />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
