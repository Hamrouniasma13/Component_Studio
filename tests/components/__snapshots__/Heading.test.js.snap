// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: <Heading /> render <h1> 1`] = `
.emotion-0 {
  display: block;
  font-family: Source Sans Pro,Arial,Helvetica,sans-serif;
  margin-bottom: 0;
  font-size: 36px;
  line-height: 1.2em;
  font-weight: 600;
}

@media only screen and (min-width :768px) {
  .emotion-0 {
    font-size: 48px;
    -webkit-letter-spacing: 1.4px;
    -moz-letter-spacing: 1.4px;
    -ms-letter-spacing: 1.4px;
    letter-spacing: 1.4px;
    line-height: 1.15em;
  }
}

<h1
  className="emotion-0"
  size={undefined}
>
  Magna cillum consectetur.
</h1>
`;
